Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP_test.py

```diff
@@ -0,0 +1,1267 @@
+        assert output['guid'] == mock_output['guid']
+        assert output['name'] == mock_output['name']
+        assert output['type'] == mock_output['type']
+
+
+@pytest.mark.parametrize(
+    'file, suffix, args',
+    [(
+        'file_list_item_list_response.json',
+        'file_lists/1/files',
+        {'file_list_guid': '1'},
+    ), (
+        'file_list_item_get_response.json',
+        'file_lists/1/files/1',
+        {'file_list_guid': '1', 'sha256': '1'},
+    )]
+)
+def test_file_list_item_list_command(requests_mock, mock_client, file, suffix, args):
+    """
+    Scenario:
+    -   Get a file item list.
+    -   Get a specific file item list item.
+    Given:
+    -   The user has entered a file_list_guid.
+    -   The user has entered a file_list_guid and a sha256.
+    When:
+    -    cisco-amp-file-list-item-list is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure there are no links in the outputs.
+    """
+    mock_response = load_mock_response(file)
+    requests_mock.get(
+        f'{BASE_URL}/{suffix}',
+        json=mock_response
+    )
+
+    from CiscoAMP import file_list_item_list_command
+    response = file_list_item_list_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.FileListItem'
+    assert 'links' not in response.outputs
+
+    if policies := response.outputs[0].get('policies'):
+        assert_output_has_no_links(policies)
+
+        for policy, mock_policy in zip(policies, mock_response['data']['policies']):
+            assert policy['guid'] == mock_policy['guid']
+            assert policy['name'] == mock_policy['name']
+
+    if items := response.outputs[0].get('items'):
+        assert_output_has_no_links(items)
+
+        for item, mock_item in zip(items, mock_response['data']['items']):
+            assert item['sha256'] == mock_item['sha256']
+            assert item['source'] == mock_item['source']
+
+
+def test_file_list_item_create_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Create an item for a file item list
+    Given:
+    -   The user has entered a file_list_guid and a sha256.
+    When:
+    -    cisco-amp-file-list-item-create is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure there are no links in the outputs.
+    """
+    args: Dict[str, Any] = {
+        'file_list_guid': '1',
+        'sha256': '1'
+    }
+
+    mock_response = load_mock_response('file_list_item_create_response.json')
+    requests_mock.post(
+        f'{BASE_URL}/file_lists/{args["file_list_guid"]}/files/{args["sha256"]}',
+        json=mock_response
+    )
+
+    from CiscoAMP import file_list_item_create_command
+    response = file_list_item_create_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.FileListItem'
+    assert 'links' not in response.outputs
+    assert response.outputs[0]['sha256'] == mock_response['data']['sha256']
```

Comment:
please compare whole outputs, not just specific fields - in all tests. I'll continue reviewing it once it's done

Reply:
Changed.